src/OpenTelemetry.Extensions.Propagators/README.md

```diff
@@ -0,0 +1,35 @@
+# Trace context propagators for OpenTelemetry .NET
+
+[![NuGet](https://img.shields.io/nuget/v/OpenTelemetry.Extensions.Propagators.svg)](https://www.nuget.org/packages/OpenTelemetry.Extensions.Propagators)
+[![NuGet](https://img.shields.io/nuget/dt/OpenTelemetry.Extensions.Propagators.svg)](https://www.nuget.org/packages/OpenTelemetry.Extensions.Propagators)
+
+The package provides context propagators (currently supporting B3 format) for tracing.
+
+## Installation
+
+```shell
+dotnet add package OpenTelemetry.Extensions.Propagators
+```
+
+## Configuration
+
+Run `B3 OpenZipkin` context only:
+
+```csharp
+Sdk.SetDefaultTextMapPropagator(new B3Propagator())
```

Comment:
Since the new and old package has the exact same class and namespace, would this cause errors? like ambiguous class?

Reply:
just realized this is totally different namespace!